includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -780,6 +780,20 @@ public function is_valid_optimized_checkout_page(): bool {
+	 * Returns the payment method title.
+	 *
+	 * When Optimized Checkout is enabled, returns the title from the OC payment method class.
+	 *
+	 * @return string
+	 */
+	public function get_title() {
+		if ( $this->oc_enabled && $this->is_valid_optimized_checkout_page() ) {
+			return ( new WC_Stripe_UPE_Payment_Method_OC() )->get_title();
+		}
+		return parent::get_title();
+	}
```

PHPMD (2.15.0)

[error] The method get_title is not named in camelCase. (undefined)

(CamelCaseMethodName)